specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -1798,6 +2706,122 @@
       },
       "x-ms-discriminator-value": "ApiKey"
     },
+    "CognitiveServiceEndpointDeploymentResourceProperties": {
+      "required": [
+        "model"
+      ],
+      "type": "object",
+      "properties": {
+        "model": {
+          "description": "Model used for the endpoint deployment.",
+          "$ref": "#/definitions/EndpointDeploymentModel"
+        },
+        "properties": {},
```

Comment:
>"properties": {}, [](http://example.com/codeflow?start=8&length=17)

This duplicates top-level property "properties" and isn't allowed inside the property bag. Also, empty/unschematized objects aren't allowed.

Reply:
removed